packages/contracts-bedrock/test/L1/SystemConfig.t.sol

```diff
@@ -227,8 +235,8 @@ contract SystemConfig_Init_CustomGasToken is SystemConfig_Init {
 
         systemConfig.initialize({
             _owner: alice,
-            _overhead: 2100,
-            _scalar: 1000000,
+            _basefeeScalar: 2100,
+            _blobbasefeeScalar: 1000000,
```

Comment:
This works, but are weird values to use for the base fee and blob base fee scalars, as they're nominally the same values as `overhead` and `scalar` before. Maybe we use the current OP mainnet values instead?

Reply:
I am down for using whatever values here, i don't think the values themselves are super important. One consideration is that it should be a fuzz test when the values aren't super important, but I thought that would be a bit overkill in this particular situation